### DISCUSSION TITLE
How do i change resolution of the textures which are being converted to .ktx2 ?

### DISCUSSION THREAD

**Discussion by vis-prime** [KhronosGroup/KTX-Software#695] [Q&A] [answered]
In blender i'm using 4k and 8k textures 

for the web I prefer it to be 2k or 1k

any way to pass a resolution in the command ?

`toktx --encode etc1s  "F:\Blends\mats list\exp/wooden_rough_planks_diff_4k"  "F:\Poly Haven\wooden_rough_planks\textures\wooden_rough_planks_diff_4k.jpg"`

**Comment by MarkCallow** [comment, marked as answer]
You can use one of:

       --resize <width>x<height>
           Resize images to width X height. This should not be used with
           --mipmap as it would resize all the images to the same size.
           Resampler options can be set via --filter and --fscale.

       --scale <value>
           Scale images by value as they are read. Resampler options can be
           set via --filter and --fscale. .
